### ISSUE TITLE
Clarify to which machines the "Connect to the BCB board automatically" section in "Bluetooth" section of icub-head refers

### ISSUE THREAD

**Issue by traversaro** [icub-tech-iit/documentation#162]
The section https://icub-tech-iit.github.io/documentation/icub_operating_systems/icubos/bluetooth/#connect-to-the-bcb-board-automatically seems to apply to any icub-head with Bluetooth capabilities, but it actually refers only to iCub 2.5/2.6/2.7 (not iCub 3) that have an actually battery backpack. All other users can safely skip that section.

**Comment by maggia80** [comment]
yes, it applies only to iCub 2.5/2.6/2.7. In iCub3 there is another board (BAT board) with a different Bluetooth module (BLE 4.0) 
The firmware for the BAT board BLE module is still not developed.
